### ISSUE TITLE
no publish volume, capacity can't be recycled

### ISSUE THREAD

**Issue by SSmallMonster** [hwameistor/hwameistor#232]
LogInfo:
```
[root@demo-dev-master-03 /etc/drbd.d]# k get lv pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
NAME                                       POOL                   REPLICAS   CAPACITY      ACCESSIBILITY   STATE   RESOURCE   PUBLISHED   AGE
pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f   LocalStorage_PoolHDD   2          53687091200                   Ready   1                      20h
[root@demo-dev-master-03 /etc/drbd.d]# k get lvr -o yaml|grep pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
    name: pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f-5zzjzw
    volumeName: pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
    devPath: /dev/LocalStorage_PoolHDD-HA/pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
    storagePath: /dev/LocalStorage_PoolHDD/pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
    name: pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f-fwmmbj
    volumeName: pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
    devPath: /dev/LocalStorage_PoolHDD-HA/pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
    storagePath: /dev/LocalStorage_PoolHDD/pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
[root@demo-dev-master-03 /etc/drbd.d]# k get lvr |grep pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f-5zzjzw   53687091200   demo-dev-worker-02   Ready      true     /dev/LocalStorage_PoolHDD-HA/pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f   20h
pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f-fwmmbj   53687091200   demo-dev-master-03   Ready      true     /dev/LocalStorage_PoolHDD-HA/pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f   20h
[root@demo-dev-master-03 /etc/drbd.d]# k get lsn demo-dev-master-03 -o yaml|grep pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
      - pvc-a1e0e098-bd43-4ee7-8ea7-f40b41ee7f9f
```

If this happens, the data volume will not be managed or released

**Comment by SSmallMonster** [comment]
You can manually change the delete field of Lv to release the volume

**Comment by sun7927** [comment]
Deadline: August

**Comment by SSmallMonster** [comment]
fix at: #299
